### PR TITLE
add extract-api.yaml

### PR DESCRIPTION
**Pull request by DanRod1999**
resolves: https://github.com/iTwin/imodel-transformer/issues/214

### CODE REVIEW

File: .github/workflows/extract-api.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        run: |
+          git config --local user.email imodeljs-admin@users.noreply.github.com
+          git config --local user.name imodeljs-admin
+
+      - name: Install pnpm
+        uses: pnpm/action-setup@v4
+        with:
+          version: 7.27.0
+
+      - name: Use Node.js 18
```
Comment (MichaelSwigerAtBentley) [inline .github/workflows/extract-api.yaml:38]: Node 18 EoL is coming up, maybe we should add an issue to update to 22

**Review by MichaelSwigerAtBentley — DISMISSED** [review]

File: .github/workflows/extract-api.yaml (hunk 1 above)
Comment (DanRod1999) [reply .github/workflows/extract-api.yaml:38]: good call. don't think this is the only occurrence. I'll check and maybe open a new pr just for the sake of easier tracking

**Comment by MichaelSwigerAtBentley** [comment]
Adding my review, Hold on merging until someone else approves as well

**Comment by DanRod1999** [comment]
> Adding my review, Hold on merging until someone else approves as well

yeah going to wait for Nick as well since this required some file name changes that may or may not be necessary

File: .github/workflows/extract-api.yaml (hunk 1 above)
Comment (MichaelSwigerAtBentley) [reply .github/workflows/extract-api.yaml:38]: Yes,18 is used across the board it seems, should be a new PR. Just wanted to bring attention to it. Resolving

**Review by nick4598 — APPROVED** [review]

**Pull request merged**